### PR TITLE
[Gekidou] Set 5s as default timeout for ping action

### PR DESCRIPTION
**Pull request by enahum**
```release-note
NONE
```

### CODE REVIEW

File: app/actions/remote/general.ts [hunk 1]
```diff
@@ -35,7 +35,8 @@ async function getDeviceIdForPing(serverUrl: string, checkDeviceId: boolean) {
     return getDeviceToken(appDatabase);
 }
 
-export const doPing = async (serverUrl: string, verifyPushProxy: boolean) => {
+// Default timeout interval for ping is 5 seconds
+export const doPing = async (serverUrl: string, verifyPushProxy: boolean, timeoutInterval = 5) => {
```
Comment (larkox) [inline app/actions/remote/general.ts:39]: On the title of the PR you say 3 seconds, but here is 5 seconds.

**Review by larkox — APPROVED** [review]
LGTM, just one minor detail.

File: app/actions/remote/general.ts (hunk 1 above)
Comment (enahum) [reply app/actions/remote/general.ts:39]: oh lol... 😂 

what you think, should we use 3 or 5?

Comment (larkox) [reply app/actions/remote/general.ts:39]: I would go with 5, but 0/5 :sweat_smile:

Comment (enahum) [reply app/actions/remote/general.ts:39]: done!

**Pull request merged**